go/enclave/crosschain/interfaces.go

```diff
@@ -0,0 +1,43 @@
+package crosschain
+
+import (
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/state"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/obscuronet/go-obscuro/go/common"
+)
+
+type (
+	OnChainEVMExecutorResponse = map[common.TxHash]interface{}
+	OnChainEVMExecutorFunc     = func(common.L2Transactions) map[common.TxHash]interface{}
+	OffChainEVMCallFunc        = func(types.Message) (*core.ExecutionResult, error)
+)
+
+type MainNetMessageExtractor interface {
+	//(todo: Verifies receipts match block) & extracts the messages logged inside the receipts
+	ProcessCrossChainMessages(block *common.L1Block, receipts common.L1Receipts) error
+
+	GetBusAddress() *common.L1Address
+
+	Enabled() bool
+}
+
+type ObscuroCrossChainManager interface {
+	// Address of the identity owning the message bus.
+	GetOwner() common.L2Address
```

Comment:
the golang convention is to start each comment of public methods with the name of the method (🤷‍♂️)